lib/tasks/import_data.rb

```diff
@@ -0,0 +1,63 @@
+require "json"
+
+module Tasks
+  class ImportData
+    def initialize(file:, total_lines:, stdout:, draft:)
+      @file = file
+      @total_lines = total_lines
+      @stdout = stdout
+      @draft = draft
+    end
+
+    def import_all
+      file.each.with_index(1) do |json, index|
+        parsed_json = JSON.parse(json).deep_symbolize_keys
+
+        updated_at = parsed_json.fetch(:updated_at)
+        content_item_hash = parsed_json.fetch(:content_item)
+        content_id = content_item_hash.fetch(:content_id)
+
+        unless Event.where("payload ->> 'content_id' = '#{content_id}'").exists?
```

Comment:
@rboulton The case this is protected against isn't bad data, but avoiding overwriting newer data.

This import will be run after these event logs and derived representations are being created via the API.  Our data exported from the content store should be by definition equal to or older than any data in the publishing API's databases, so we at best don't need to save it, and at worst really don't want to.